src/TensorFlow.jl

```diff
@@ -165,14 +165,13 @@ library loaded.
 macro py_proc(expr)
     quote
         eval(Main, quote
-            remotecall_fetch($(TensorFlow.load_python_process())) do
+            remotecall_wait($(TensorFlow.load_python_process())) do
```

Comment:
Because it is mutating state?
Ok.

So what is the point of doing a `wait` rather than a `fetch` here?
The time taken will be very similar.
Are you just trying to keep the stack-trace nice, for when exceptions are thrown?